tests/unit/ListPeopleAndRoles.spec.ts

```diff
@@ -0,0 +1,244 @@
+
+    expect(peoplesListItem1.querySelectorAll('.col-roles')[0].textContent)
+      .toContain('Completing Party')
+    expect(peoplesListItem1.querySelectorAll('.col-roles')[1].textContent)
+      .toContain('Director')
+
+    expect(peoplesListItem2.querySelector('.col-roles').textContent).toContain('Incorporator')
+  })
+
+  it('displays the actions menu when viewed not in summary view', () => {
+    const wrapper = wrapperFactory({ personList: mockPersonList })
+    const peoplesListItem1 = wrapper.vm.$el.querySelectorAll('.people-roles-content')[0]
+    const peoplesListItem2 = wrapper.vm.$el.querySelectorAll('.people-roles-content')[1]
+
+    expect(peoplesListItem1.querySelector('.actions')).not.toBeNull()
+    expect(peoplesListItem2.querySelector('.actions')).not.toBeNull()
+  })
+
+  it('does NOT display the actions menu when viewed in summary view', () => {
+    const wrapper = wrapperFactory({ personList: mockPersonList, isSummary: true })
+    const peoplesListItem1 = wrapper.vm.$el.querySelectorAll('.people-roles-content')[0]
+    const peoplesListItem2 = wrapper.vm.$el.querySelectorAll('.people-roles-content')[1]
+
+    expect(peoplesListItem1.querySelector('.actions')).toBeNull()
+    expect(peoplesListItem2.querySelector('.actions')).toBeNull()
+  })
+
+  it('displays the Summary header when in summary view', () => {
+    const wrapper = wrapperFactory({ personList: mockPersonList, isSummary: true })
+
+    expect(wrapper.vm.$el.querySelector('.people-roles-summary-header').textContent)
+      .toContain('People and Roles')
+  })
+
+  it('does NOT display the Summary header when NOT in summary view', () => {
+    const wrapper = wrapperFactory({ personList: mockPersonList })
+
+    expect(wrapper.vm.$el.querySelector('.people-roles-summary-header')).toBeNull()
+  })
+
+  it('displays invalid warning message when in summary view and step 2 data is invalid', () => {
+    const wrapper = wrapperFactory({ personList: mockPersonList, showErrorSummary: true, isSummary: true })
+
+    expect(wrapper.vm.$el.querySelector('.people-roles-invalid-message').textContent)
+      .toContain('This step is not complete.')
+
+    expect(wrapper.vm.$el.querySelector('.people-roles-invalid-message').textContent)
+      .toContain('Return to this step to complete it.')
+  })
+
+  it('sends you to step 2 when the error message link is clicked', () => {
+    const localVue = createLocalVue()
+    localVue.use(VueRouter)
+    const router = mockRouter.mock()
+    const wrapper = mount(ListPeopleAndRoles,
+      {
+        propsData: { personList: mockPersonList, showErrorSummary: true, isSummary: true
+        },
+        vuetify,
+        localVue,
+        router
+      })
+    expect(wrapper.vm.$route.name).toBeNull()
+
+    const errorLink = wrapper.find('#router-link')
+    errorLink.trigger('click')
+
+    expect(wrapper.vm.$route.name).toBe('add-people-roles')
+  })
+})
```

Comment:
Should also have tests for roles (ie, ensure correct roles are displayed; ensure Add Role gets displayed).

Reply:
As per discussions, we will expand on the unit testing in the next pr! 👍